README.md

```diff
@@ -1,64 +1,56 @@
-5. Set the _LinkedRole1_ and _LinkedRole2_ parameters to any additional end user roles you may want to link to the Portfolio.
-6. Set the _CreateEndUsers_ parameter to No if you have already run a Portfolio stack from this repo (ServiceCatalogEndusers already exists).
-7. Change the _RepoRootURL_ parameter to your bucket's root url:  
-  ```https://s3.amazonaws.com/[YOUR-BUCKET-NAME-HERE]/```  
-  
-### AWS Service Catalog Product Launch  
-After creating the Cloudformation Stacks you will have a ServiceCatalog Portfolio with products, launch constraints, and associated user roles.  Your end users may now launch products from the ServiceCatalog dashboard. To learn more about the end user dashboard for AWS Service Catalog, see [Using the End User Console View](https://docs.aws.amazon.com/servicecatalog/latest/userguide/end-user-console.html)
-
-![sc-ra-products.png](sc-ra-products.png)
-
-## Warning
-Before you distribute the CloudFormation template to your organization, review the template. 
-Check IAM permissions, Deletion policies, update stack behavior, other aspects of the template, 
-and ensure that they are as per your expectations and processes. 
-These sample CloudFormation templates may need updates before you can use them in production.  
-Running these templates may result in charges to your AWS account.  
-Provisioning the supplied Products through ServiceCatalog will create AWS Services which will be billed to your account.
-
+ Read the [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-reference-servicecatalog.html) for more information on ServiceCatalog and CloudFormation.  
+
+Sage Bionetworks uses a second repository, [scipoolprod-infra](https://github.com/Sage-Bionetworks/scipoolprod-infra), to deploy Cloudformation stacks defined by the templates in this library.
+
+## Deployment
+Merges to master will copy these templates to an S3 bucket that is used by [scipoolprod-infra](https://github.com/Sage-Bionetworks/scipoolprod-infra) to deploy stacks.
+
+## Development
+Currently there is a manual process for development. The script `dev-deploy-templates.sh`, when run from the project root, takes a file or directory path and copys the templates found there to the deployment bucket. These can then be referenced when creating deployment stacks in [scipoolprod-infra](https://github.com/Sage-Bionetworks/scipoolprod-infra).
+
+### Account and Credentials
+While this library could be used in any AWS account, Sage Bionetworks is currently deploying to the "scipoolprod" account. In order to have permission to run the `dev-deploy-templates.sh` script, use the `AWSIAMCfServiceRole` role or something with similar permissions. Ask for guidance on this point from #sageit Slack channel.
+
+In the second half of the development process, when stacks are deployed from `scipoolprod-infra`, one needs access to the `AWSIAMAdminRole`. 
```

Comment:
this also uses the `AWSIAMCfServiceRole`

Reply:
No, not really. Remember this is specifically a _development_ guide. This has nothing to do with explaining how the CICD system works.